roles/keycloak/defaults/main.yml

```diff
@@ -0,0 +1,336 @@
+---
+# if the keycloak archive is from the "stage" branch, then you might enable the wayf plugin (keycloak-theme-vanilla)
+keycloak_archive_url: "https://github.com/keycloak/keycloak/releases/download/13.0.0/keycloak-13.0.0.tar.gz"
```

Comment:
Replace with "https://github.com/keycloak/keycloak/releases/download/{{ keycloak_version }}/keycloak-{{ keycloak_version }}.tar.gz" and to improve readability define keycloak_version before  keycloak_archive_url